list.md

```diff
@@ -7,7 +7,7 @@ librarians.
 Library institutions, journals, and projects
 ---------------------------------------------
 
-SUB Göttingen
+Niedersächsische Staats- und Universitätsbibliothek Göttingen (SUB Göttingen, SUBUGOE)
```

Comment:
I'd omit _SUBUGOE_ (or is that something used outside of GitHub)?

In addition, I suggest to squash all commits (combine them in a single commit).